cfgen/cfgen

```diff
@@ -831,22 +836,25 @@ def oids_to_dhall(oids: List[Oid]) -> str:
 NetProtocol = Enum('NetProtocol', 'tcp o2ib')
 
 
-ProcT = Enum('ProcT', 'hax m0_server m0_client_s3 m0_client_other')
+ProcT = Enum('ProcT', 'hax ios client_s3 m0server_confd client_other cas')
 ProcT.__doc__ = 'Type of process'
 
 
 class LnetPortalGroup(IntEnum):
     hax = 1,
-    m0_server = 2,
-    m0_client_s3 = 3,
-    m0_client_other = 4
+    ios = 2,
+    client_s3 = 3,
+    client_other = 4
 
 
 class LibfabPortalGroup(IntEnum):
-    hax = 2000,
-    m0_server = 3000,
-    m0_client_s3 = 4000,
-    m0_client_other = 5000
+    hax = 21000,
+    ios = 22000,
```

Comment:
I don't think we need different port ranges for confd, cas and ios, there should a port range for motr-servers, hare and motr-clients. Just check with the provisioner.